src/Routes.js

```diff
@@ -46,102 +73,78 @@ const Templates = lazy(() =>
         /* webpackChunkName: "PackageDetail" */ './SmartComponents/PatchSet/PatchSet'
     )
 );
-export const paths = {
-    advisories: {
-        title: 'Applicable advisories',
-        to: '/advisories'
+
+export const paths = [
+    {
+        path: '/advisories',
+        isExact: true,
+        requiredPermissions: ['patch:*:read'],
+        component: Advisories
     },
-    systems: {
-        title: 'Systems',
-        to: '/systems/'
+    {
+        path: '/systems/',
+        isExact: true,
+        requiredPermissions: ['patch:*:read'],
+        component: Systems
     },
-    inventoryDetail: {
-        title: 'Inventory detail',
-        to: '/systems/:inventoryId'
+    {
+        path: '/systems/:inventoryId',
+        isExact: true,
+        requiredPermissions: ['patch:*:read'],
+        component: InventoryDetail
     },
-    advisoryDetail: {
-        title: 'Advisory detail',
-        to: '/advisories/:advisoryId'
+    {
+        path: '/advisories/:advisoryId',
+        isExact: true,
+        requiredPermissions: ['patch:*:read'],
+        component: AdvisoryPage
     },
-    advisoryDetailSystem: {
-        title: '',
-        to: '/advisories/:advisoryId/:inventoryId'
+    {
+        path: '/advisories/:advisoryId/:inventoryId',
+        isExact: true,
+        requiredPermissions: ['patch:*:read'],
+        component: InventoryDetail
     },
-    packages: {
-        title: 'Packages',
-        to: '/packages'
+    {
+        path: '/packages',
+        isExact: true,
+        requiredPermissions: ['patch:*:read'],
+        component: PackagsPage
     },
-    packageDetail: {
-        title: 'Package detail',
-        to: '/packages/:packageName'
+    {
+        path: '/packages/:packageName',
+        isExact: true,
+        requiredPermissions: ['patch:*:read'],
+        component: PackageDetail
     },
-    packageDetailSystem: {
-        title: '',
-        to: '/packages/:packageName/:inventoryId'
+    {
+        path: '/packages/:packageName/:inventoryId',
+        isExact: true,
+        requiredPermissions: ['patch:*:read'],
+        component: InventoryDetail
     },
-    templates: {
-        title: 'Templates',
-        to: '/templates'
+    {
+        path: '/templates',
+        isExact: true,
+        requiredPermissions: ['patch:*:read'],
+        component: Templates
     }
-};
-
-export const Routes = (props) => {
-
-    const path = props.childProps.location.pathname;
-
-    const isPatchSetEnabled = useFeatureFlag(featureFlags.patch_set);
```

Comment:
Are we safe to remove the flag already? if not it is better to leave it for now.

Reply:
Ahh this was a mistake, ill add this back